### PR TITLE
Fix issue the when compiling tests in a project created by the init command which has a main and a service in the same package

### PR DESCRIPTION
**Pull request by NatashaWso2**
Fixes https://github.com/ballerina-platform/ballerina-lang/issues/9685

### CODE REVIEW

File: cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/init/models/SrcFile.java [hunk 1]
```diff
@@ -124,15 +124,15 @@
                                                         "   Before Suite Function\n" +
                                                         "}\n" +
                                                        "@test:BeforeSuite\n" +
-                                                       "function beforeSuiteFunc () {\n" +
+                                                       "function beforeSuiteServiceFunc () {\n" +
                                                        "    io:println(\"I'm the before suite function!\");\n" +
```
Comment (hemikak) [inline cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/init/models/SrcFile.java]: Shall we update the println messages as well ?

File: cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/init/models/SrcFile.java [hunk 2]
```diff
@@ -141,7 +141,7 @@
                                                        "   After Suite Function\n" +
                                                        "}\n" +
                                                        "@test:AfterSuite\n" +
-                                                       "function afterSuiteFunc () {\n" +
+                                                       "function afterSuiteServiceFunc () {\n" +
                                                        "    io:println(\"I'm the after suite function!\");\n" +
```
Comment (hemikak) [inline cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/init/models/SrcFile.java]: Shall we update the println messages as well ?

Comment (NatashaWso2) [reply cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/init/models/SrcFile.java]: Updated message with 009fc6d

File: cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/init/models/SrcFile.java (hunk 1 above)
Comment (NatashaWso2) [reply cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/init/models/SrcFile.java]: Updated message with 009fc6d

**Pull request merged**